pkgs/applications/audio/lmms/default.nix

```
@@ -52,6 +52,6 @@ mkDerivation rec {
     homepage = "https://lmms.io";
     license = licenses.gpl2Plus;
     platforms = [ "x86_64-linux" "i686-linux" ];
-    maintainers = with maintainers; [ goibhniu ];
+    maintainers = with maintainers; [ yana ];
```

Of-borg can't find a maintainer under the name yana
```suggestion
    maintainers = with maintainers; [  ];
```

---

See 39ddcbfc694c254f1af2dbd6519c7f0bf7bba055

Slipped when rebasing.

---

Fixed.